lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -201,7 +206,8 @@ def self.go_template_for(kind, name)
           %[(eq .involvedObject.kind "#{kind}")],
           %[(eq .involvedObject.name "#{name}")],
           '(ne .reason "Started")',
-          '(ne .reason "Created")'
+          '(ne .reason "Created")',
+          '(ne .reason "SuccessfulCreate")'
```

Comment:
This filters out these event types. Looking at the test output (see the screenshots), I'm thinking perhaps we should filter out `Scheduled`, `Pulling` and `Pulled` as well. WDYT? Now that pull errors should be reported, could those events be useful?